### ISSUE TITLE
Object Detection with pre trained pascal model returns 0 detections

### ISSUE THREAD

**Issue by brunovollmer** [Microsoft/CNTK#2592]
Hello everybody,

I'm currently trying to make the object detection evalutation (https://github.com/Microsoft/CNTK/blob/master/Examples/Image/Detection/FastRCNN/BrainScript/CNTK_FastRCNN_Eval.ipynb) run with the pretrained Fast-RCNN-Pascal model (https://www.cntk.ai/Models/FRCN_Pascal/Fast-RCNN.model). I changed the number of rois to 4000 and changed some parameters (min. and max relative of a roi and min. and max. relative width/height of a roi) back to the suggested parameters in FastRCNNconfig.py. 
Every time I run the script it computes some rois:

> Number of rois detected using selective search: 288
> Number of rois on grid added: 92984
> Number of rectangles before filtering  = 93272
> Number of rectangles after filtering  = 92345
> Number of rois for evaluation: 4000

But I detects nothing:

>Number of detections: 0
>Indices of selected regions: []

I tested it with different test images but all return the same 0 detections. I'm not sure if the problem is with the script or the model because when I run it with the grocery model it works as it should.

If any body has an idea what I might miss I would be happy about suggestions.

**Comment by DHOFM** [comment]
Hi,
the training script uses precomputedROIs in matlab files, so if you use different ROIs it can happen, that nothing is detected because of not matching Regions. I had the same problem and discussed it here; https://github.com/CatalystCode/CNTK-FastRCNNDetector/issues/1

**Comment by brunovollmer** [comment]
Hey @DHOFM,

thanks for the quick reply. With your changed parameters some images but return results but most return 0 detections. Isn't there a more robust way to generate ROIs for a new image?

**Comment by DHOFM** [comment]
@brunovollmer The only robust way i could think of would be to define your own ROIs and retrain the whole Pascal Data Set with these. As mentioned also by @pkranen they do not know the original parameters. after making some more changes i found a good setting for my usecase. But i only needed a car detection so if you need other labels thy could be worse. Maybe you will check out Faster R CNN which uses a RPN - i did not have the time to chack it out at the moment, but the Fast R Solution has the problem of to much time consumption, because the Selective Search Method only uses CPU and is to slow for a real time detection

**Comment by spandantiwari** [comment]
FWIW I too think that using FasterRCNN would be the better choice if at all possible. Please re-open this issue, or create a new issue, if you run into problems.
